app/views/brexit_checker/_results_business_actions.html.erb

```diff
@@ -0,0 +1,27 @@
+<% heading = t('brexit_checker.results.audiences.business.heading') %>
```

Comment:
Could you just use the translation directly without defining a variable?

Reply:
even if it's re-used in two places?
It made line 6 very long amongst the other attributes

Reply:
I think it is already DRY, because if I change the translation it changes it all usages of it. It just would take me more time to figure out where this ```heading``` variable is defined - i would assume it would have been passed in which in this case it is not.